js/index.js

```diff
@@ -40,3 +40,127 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+//logo.setAttribute("src", "img/logo.png")
+logo.src = "img/logo.png"
+
+const navBar = document.querySelectorAll(".container header nav a")
+navBar[0].textContent = "Services"
+navBar[1].textContent = "Product"
+navBar[2].textContent = "Vision"
```

Comment:
You should be pulling all of this data down from the JSON `siteContent` above. The whole things should be made as such pulling the data down from above not hard coding it, the above data is to mimic data coming in from an API.